code/modules/sanity/inspiration_component.dm

```diff
@@ -42,10 +52,29 @@
 			else
 				continue
 		to_chat(user, SPAN_NOTICE("This item has [aspect] aspect of [stat]"))
+	var/strength
```

Comment:
this is a personal opinion but I think adding this feedback is unnecessary. letting the players know if it has the mec stat and having your machine indicate that it requires an item of that type is sufficient.